opentech/apply/funds/templates/funds/applicationsubmission_detail.html

```diff
@@ -13,7 +13,7 @@
         <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
         <h5 class="heading heading--meta">
             <span>{{ object.stage }}</span>
-            <span>{{ object.page }}</span>
+            <span><a class="link--transparent link--underlined" href="{% pageurl public_page %}" >{{ object.page }}</a></span>
```

Comment:
can you not just `{% pageurl object.page %}`?

Reply:
That links to the form and they want the public page for the form.